content/c-sharp/concepts/operators/operators.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: 'Operators'
+Description: 'Operators are used to perform various operations on variables and values.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Operators'
+  - 'Arithmetic'
+  - 'Comparison'
+  - 'Logical'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+**Operators** are used to perform various operations on variables and values. The standard arithmetic and assignment operators are the most familiar.
+
+## Syntax
+
+The following code snippet uses the assignment operator, `=`, to set `my_variable` to the value of `num1` and `num2` with an arithmetic operator acting on them. For example, if `operator` represented `*`, `my_variable` would be assigned a value of `num1 * num2`.
+
+```pseudo
+my_variable = num1 operator num2;
+```
+
+Operators can be organized into the following groups:
+
+- Arithmetic operators for performing traditional math evaluations.
+- Assignment operators for assigning values to variables.
+- Comparison operators for comparing two values.
+- Logical operators for combining Boolean values.
+- Bitwise operators for manipulating the bits of a number
```

Comment:
Can we link https://www.codecademy.com/resources/docs/general/bitwise-operator here?